microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,128 @@
+Screenshot 1, "after" in Edge 105: Ensure Focus Mode is enabled. Open Settings menu to show DevTools docking location and Activity Bar location settings. -->
+
+You can now change the orientation of the Quick View panel as well. To change Quick View to a vertical split, select the location toggle (**Dock Quick View to the right**). Selecting this toggle a second time will return Quick View to a horizontal split.
+
+![2](./devtools-105-images/quickview-console.msft.png) 
+<!-- 
+Screenshot 2: Ensure Focus Mode is enabled. With Elements open in main panel and Console in Quick View, press the location toggle at the top right of Quick View to change its orientation to vertical. -->
+
+To minimize Quick View, use the **Collapse** toggle or press `Escape`:
+
+![3](./devtools-105-images/focus-mode-improved-location-controls3.png)
+<!-- Screenshot 3: Continuation of previous screenshot. Minimize Quick View in vertical orientation with with Esc keyboard shortcut. See below. -->
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## Improved reliability in Issues tool
+<!-- Rank 6 -->
+<!-- Title: Improved reliability in Issues tool -->
+<!-- Subtitle: Improved reliability in Issues tool. -->
+
+In previous versions of Microsoft Edge, there was a bug that sometimes resulted in issues not appearing in the **Issues** tool.  This bug has been fixed, resulting in improved reliability of the **Issues** tool.
+
+![entry 2](./devtools-105-images/pngtemplate.png)
+
+<!-- See also:
+* []()
+* []() -->
+
+
+<!-- ====================================================================== -->
+<!-- ## 3 -->
+
+<!-- Title:  -->
+<!-- Subtitle: . -->
+
+
+<!-- ![3](./devtools-105-images/pngtemplate.png) -->
+
+<!-- See also:
+* []()
+* []() -->
+
+
+<!-- ====================================================================== -->
+<!-- ## 4 -->
+
+<!-- Title:  -->
+<!-- Subtitle: . -->
+
+
+<!-- ![4](./devtools-105-images/pngtemplate.png) -->
+
+<!-- See also:
+* []()
+* []() -->
+
+
+<!-- ====================================================================== -->
+## Announcements from the Chromium project
+
+Microsoft Edge 105 also includes the following updates from the Chromium project:
+
+* [Step-by-step replay in the Recorder](https://developer.chrome.com/blog/new-in-devtools-105/#recorder)
+* [Support mouse over event in the Recorder panel](https://developer.chrome.com/blog/new-in-devtools-105/#recorder-hover)
+* [Largest Contentful Paint (LCP) in the Performance insights panel](https://developer.chrome.com/blog/new-in-devtools-105/#lcp)
+* [Identify flashes of text (FOIT, FOUT) as potential root causes for layout shifts](https://developer.chrome.com/blog/new-in-devtools-105/#foit-fout)
+* [Protocol handlers in the Manifest pane](https://developer.chrome.com/blog/new-in-devtools-105/#manifest)
```

Comment:
```suggestion
```
@captainbrosset - I moved your comment here to /09/ dir's .md.  Do you specifically mean remove "* [Protocol handlers in the Manifest pane](https://developer.chrome.com/blog/new-in-devtools-105/#manifest)"?

Reply:
Yes, since we developed this feature, it should have its own separate announcement as a h2 header (like "Improved reliability in Issues tool"), instead of being lost in the Chromium project announcements.